src/coreComponents/physicsSolvers/fluidFlow/SinglePhaseHybridFVM.hpp

```diff
@@ -164,8 +166,6 @@ class SinglePhaseHybridFVM : public SinglePhaseBase
                              real64 const & dt,
                              DomainPartition & domain ) override;
 
-  virtual void updatePressureGradient( DomainPartition & domain ) override final;
```

Comment:
Was this function just not used?

Reply:
this is used for the phase field runs I think. @frankfeifan ?

Reply:
it is only implemented in `SinglePhaseHybridFVM` and is not called anywhere, but maybe i am missing something

Reply:
It's the first part of [this](https://github.com/GEOS-DEV/GEOS/pull/2125/files#diff-5baed9381deba141da8e1689df8266136f85a35a73e53499931634d0148025af) work that was merged. we have not merged the rest yet but that function is needed.

Reply:
it should be restored soon by https://github.com/GEOS-DEV/GEOS/pull/3472
sorry for my misunderstanding